### PR TITLE
152670109 Better service search page

### PR DESCRIPTION
**Pull request by tatut**
A new custom search page that can be used instead of the
CKAN default dataset page.

Supports our two facets: service type and operation area as well as free text search on name.

### CODE REVIEW

File: ote/resources/public/language/fi.edn [hunk 1]
```diff
@@ -104,7 +105,11 @@
   ;; Common fields
   :ote.db.common/street                               "Osoite"
   :ote.db.common/post_office                          "Postitoimipaikka"
-  :ote.db.common/postal_code                          "Postinumero"}
+  :ote.db.common/postal_code                          "Postinumero"
+
```
Comment (rckopteri) [inline ote/resources/public/language/fi.edn]: Not a biggie, but these empty rows are not necessary.

File: ote/src/clj/ote/services/service_search.clj [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            [jeesql.core :refer [defqueries]]
+            [ote.db.transport-service :as t-service]
+            [ote.db.service-search :as search]
+            [specql.op :as op]
+            [clojure.set :as set]
+            [clojure.string :as str]))
+
+(defqueries "ote/services/service_search.sql")
+
+(defn search-facets [db]
```
Comment (rckopteri) [inline ote/src/clj/ote/services/service_search.clj]: Where this is used and what it does?

File: ote/src/cljs/ote/views/service_search.cljs [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      [(form/group
+        {:label (tr [:service-search :filters-label])
+         :columns 3
+         :layout :row}
+
+        {:name :text-search
+         :type :string
+         :placeholder (tr [:service-search :text-search-placeholder])}
+
+        ;; ekaksi tämä
```
Comment (rckopteri) [inline ote/src/cljs/ote/views/service_search.cljs]: translate

File: ote/src/cljs/ote/views/service_search.cljs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            [cljs-react-material-ui.reagent :as ui]
+            [ote.app.controller.service-search :as ss]))
+
+
+(defn results-listing [e! results]
+  [ui/grid-list
+   (doall
+    (for [{::t-service/keys [id name] :as service} results]
+      [ui/grid-tile {:key id}
+       [:div "resultti on " (pr-str service)]]))])
```
Comment (mmarkuss) [inline ote/src/cljs/ote/views/service_search.cljs]: small thing. div name in english.

**Pull request merged**